home-data-migration/src/main/resources/db/changelog/v-0.0.1/001-changeset-users-table.xml

```diff
@@ -0,0 +1,170 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+
+    <changeSet id="1" author="Arthurpanda">
```

Comment:
Let's link the id of the changeset to the issue so it will be clear in the scope of what issue this change was created:
`id="1"` -> `id="Issue33.001"`

Reply:
Renamed